themes/default/content/blog/top-5-things-for-azure-devs-serverless/index.md

```diff
@@ -1,18 +1,19 @@
 ---
 title: "Top 5 an Azure Developer Needs to Know: Serverless"
-date: 2021-07-30
-meta_desc: "Building and deploying serverless functions for Azure developers"
+date: 2021-08-06
```

Comment:
If the date is in the future, does it publish on that date like most standard Hugo installs?

Reply:
It kinda does, there's a convoluted explanation [here](https://github.com/pulumi/pulumi-hugo/blob/master/BLOGGING.md#a-note-on-dates-and-scheduling-for-future-publishing). I just merge on the day of to avoid all the editing.

Reply:
Ah, makes sense. Thanks!